### ISSUE TITLE
Black screen after take/choose a photo 

### ISSUE THREAD

**Issue by tuanhoang97** [DroidNinja/Document-Scanner#10]
Hi,
I clone your project, import to android studio and run it up, everything work fine
but, after i choose a photo from gallery or take a photo, next screen is black, no polygon view, not thing!

**Comment by vamsee9** [comment]
yeah! I think you need to fix some files import activity from scanlibrary like (R.layout.scanfragmentlayout) in this you need to add some XML stuff from this [link](https://github.com/jhansireddy/AndroidScannerDemo/blob/master/ScanDemoExample/scanlibrary/src/main/res/layout/scan_fragment_layout.xml)

**Comment by ando0689** [comment]
I have the same problem, although when I download the app from play store it works fine ..
